datashader/spatial/viewshed.py

```diff
@@ -1,30 +1,23 @@
 import xarray
 import numpy as np
-import math
-
-from math import atan, sqrt, fabs
-from math import pi as PI
-
 import numba as nb
 from numba import jit
+from math import atan, sqrt, fabs
+from math import pi as PI
```

Comment:
Usually imports should be in order from least to most specific, so stdlib imports like those from `math` here should move _above_ those from `numba`.

Reply:
In fact those should go at the very top, above xarray and numpy as well.

Reply:
@jbednar done